data/human/human missions.txt

```diff
@@ -5428,3 +5428,76 @@ mission "Deal Change 2"
 				die
 			label skip
 			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`
+
+
+
+mission "Quicksilver Mixup [2]"
+	name "Misposted mail delivery"
+	description "Carry misposted mail from Silver to Quicksilver by <date> and then come back with the return shipment"
+	minor
+	deadline 10 1
+	source "Silver"
+	destination "Quicksilver"
+	cargo "misposted mail" 38
+	to offer
+		has "Quicksilver Mixup [1]: done"
+		has "quicksilver mixup 2 available"
+	on offer
+		conversation
+			`As you walk through Silver's spaceport, somebody wearing office attire and carrying a tablet power walks up to you after ending a hurried conversation with another pilot.`
+			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to Quicksilver and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day>? We'll pay you what he'd be paid, <payment>."`
+			choice
+				`	"Oh, so you guys did set up that regular shipment after all!"`
+					goto context
+
+				`	"Sure, how much mail are we talking about?"`
+					goto shipment
+
+				`	"Sorry, but I'm not heading in that direction."`
+					goto refuse
+
+			label context
+			`	The mail employee appears confused for a moment, leaning his head to the side slightly.
```

Comment:
```suggestion
			`	The mail employee appears confused for a moment, leaning his head to the side slightly.`
```
Missing quote. If you're putting together more missions in the future, I'd definitely recommend Visual Studio Code with the Endless Sky extension. That makes it pretty apparent when quotes aren't where they need to be.
![image](https://user-images.githubusercontent.com/4471575/218886304-543be4d2-7cea-45a8-bdb5-c4b5aab616b9.png)
![image](https://user-images.githubusercontent.com/4471575/218886332-5724cba9-a4c9-435f-a444-f5206d5a51b3.png)

Reply:
Hahaha that was me, I forgot it while doing the suggestion : )